src/signals/incident/components/form/MapSelectors/Streetlight/StreetlightLayer/__tests__/StreetlightLayer.test.tsx

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2021 Gemeente Amsterdam
+import type { FeatureCollection } from 'geojson'
+
+import { render, screen } from '@testing-library/react'
+import { Map } from '@amsterdam/react-maps'
+import MAP_OPTIONS from 'shared/services/configuration/map-options'
+
+import streetlightsJson from 'utils/__tests__/fixtures/streetlights.json'
+import { wegenVerkeerStraatmeubilair } from 'signals/incident/definitions/wizard-step-2-vulaan/wegen-verkeer-straatmeubilair'
+import { WfsDataProvider } from 'signals/incident/components/form/MapSelectors/Asset/Selector/WfsLayer/context'
+import type { AssetSelectValue } from 'signals/incident/components/form/MapSelectors/Asset/types'
+import {
+  contextValue,
+  withAssetSelectContext,
+} from 'signals/incident/components/form/MapSelectors/Asset/__tests__/context.test'
+import StreetlightLayer from '../StreetlightLayer'
+
+const { meta } = wegenVerkeerStraatmeubilair.extra_straatverlichting_nummer
+const assetSelectProviderValue: AssetSelectValue = {
+  ...contextValue,
+  selection: [
+    {
+      id: '031346',
+      type: '4',
+      description: 'Overig lichtpunt',
+      isReported: true,
+    },
+    {
+      id: '27235',
+      type: '4',
+      description: 'Overig lichtpunt',
+      isReported: false,
+    },
+  ],
+  meta,
+}
+
+describe('StreetlightLayer', () => {
+  const updateSpy = jest.fn()
+  const withMapStreetlight = () =>
+    withAssetSelectContext(
+      <Map data-testid="map-test" options={MAP_OPTIONS}>
+        <WfsDataProvider value={streetlightsJson as FeatureCollection}>
+          <StreetlightLayer />
+        </WfsDataProvider>
+      </Map>,
+      { ...assetSelectProviderValue, update: updateSpy }
+    )
+
+  it('should render the streetlight layer in the map', () => {
+    render(withMapStreetlight())
+    expect(screen.getByTestId('map-test')).toBeInTheDocument()
```

Comment:
I'm not seeing this data attribute defined in the `StreetlightLayer` component. Can it be set there? This assertion now depends on a value that is set somewhere outside of the scope of the unit under test, which shouldn't the case. Only attributes and values that under the direct influence of the unit under test should be asserted.

Reply:
I see your point. We can do that once we remove the cluster layer from the asset layer